Src/FluentAssertions/Formatting/DefaultValueFormatter.cs

```diff
@@ -72,10 +80,35 @@ private static bool HasDefaultToStringImplementation(object value)
     private void WriteTypeAndMemberValues(object obj, FormattedObjectGraph formattedGraph, FormatChild formatChild)
     {
         Type type = obj.GetType();
-        formattedGraph.AddLine(TypeDisplayName(type));
-        formattedGraph.AddLine("{");
+        WriteTypeName(formattedGraph, type);
+        WriteTypeValue(obj, formattedGraph, formatChild, type);
```

Comment:
⛏️ I would expect some consistency in the order of the parameters across the two calls.

Reply:
There are five methods that deal with the formatted graph:

```csharp
    private void WriteTypeAndMemberValues(object obj, FormattedObjectGraph formattedGraph, FormatChild formatChild)
    private void WriteTypeValue(object obj, FormattedObjectGraph formattedGraph, FormatChild formatChild, Type type)
    private static void WriteMemberValues(object obj, MemberInfo[] members, FormattedObjectGraph formattedGraph, FormatChild formatChild)
    private static void WriteMemberValueTextFor(object value, MemberInfo member, FormattedObjectGraph formattedGraph, FormatChild formatChild)

    private void WriteTypeName(FormattedObjectGraph formattedGraph, Type type)
```

`WriteTypeName` is the only one that doesn't require all the arguments - but it does keep the arguments it needs in a consistent order as the other methods. 😃 

I played around a bit and came up with the following change to all the method signatures that would keep them consistent and also put the two parameters used by `WriteTypeName` first:

```csharp
    private void WriteTypeAndMemberValues(FormattedObjectGraph formattedGraph, FormatChild formatChild, object obj)
    {
        Type type = obj.GetType();
        WriteTypeName(type, formattedGraph);
        WriteTypeValue(type, formattedGraph, formatChild, obj);
    }

    private void WriteTypeName(Type type, FormattedObjectGraph formattedGraph)
    private void WriteTypeValue(Type type, FormattedObjectGraph formattedGraph, FormatChild formatChild, object obj)
    private static void WriteMemberValues(FormattedObjectGraph formattedGraph, FormatChild formatChild, object obj, MemberInfo[] members)
    private static void WriteMemberValueTextFor(FormattedObjectGraph formattedGraph, FormatChild formatChild, object value, MemberInfo member)
```

I'm not entirely sure it's worth it just to have the two calls to `WriteTypeName` and `WriteTypeValue` have their common arguments first.  What do you think?

Reply:
Nah, never mind. A :pick is me nitpicking